src/mlpack/tests/random_forest_test.cpp

```diff
@@ -149,7 +149,7 @@ BOOST_AUTO_TEST_CASE(UnweightedNumericLearningTest)
   size_t rfCorrect = arma::accu(rfPredictions == testLabels);
   size_t dtCorrect = arma::accu(dtPredictions == testLabels);
 
-  BOOST_REQUIRE_GE(rfCorrect, dtCorrect);
+  BOOST_REQUIRE_GE(rfCorrect, dtCorrect * 0.85);
```

Comment:
Did the performance in terms of accuracy drop with the changes?

Reply:
Using the sample app, I noticed the accuracy dropped around 10% in average... not sure about this test

Reply:
Thanks for the input, didn't expect to see 100% accuracy; I guess if we tweak the model parameter we can probably get the same result.

Reply:
Yeah, this was a bit fishy, and after a lot of investigation my conclusions:

 * The last change actually did introduce some problems.  I hope they are fixed now.
 * I'm not sure why the original tests did not fail.  I think it is a "lucky bug".  It's not, in general, guaranteed that a random forest will outperform a single decision tree.  It's very likely, but like I said, not guaranteed.  So I can safely set the tolerance at `0.9 * dtCorrect` and I have no problems when I run many trials.